glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -519,6 +520,14 @@ open class GleanInternalAPI internal constructor () {
             return@launch
         }
 
+        if (pingNames.contains("metrics")) {
```

Comment:
Since we also support custom pings that might contain the `metrics` word (or baseline), can we make this an equality?

Reply:
`pingNames` is an array of strings, so checking if it contains `"metrics"` is already an equality check (that is: it would fail for `pingNames = ["my-metrics-things"]`)

Reply:
`pingNames` is a collection so using `.contains()` is checking for an element that exactly matches "metrics"

Reply:
My bad!